integration_tests/suite/test_bus.py

```diff
@@ -246,6 +233,7 @@ async def _connect(
             yield
         finally:
             self.auth_client.revoke_token(token)
+            await self.websocketd_client.close()
```

Comment:
lol avoid this kind of fix inside another commit
It's really usefull to know why you add it and what it fixes

Reply:
To await :)